### PR TITLE
Adding config info to allow urls with ports

### LINKED ISSUE #103: Docs-scraper not working when url has a port
As for now, docs-scraper does not work on docs-scraper. A workaround would be to use a tool like [ngrok](https://ngrok.com/).

> Ngrok exposes local servers behind NATs and firewalls to the public internet over secure tunnels

The bug should be fixed.

When trying to scrap with localhost you will receive the following result: 

> Docs-Scraper: http://localhost:8080 0 records)

Problem existed in docsearch-scraper as well: https://github.com/algolia/docsearch-scraper/issues/461#issuecomment-496581266

### PR DESCRIPTION
**Pull request by sanders41**
Closes #103

### CODE REVIEW

**Review by curquiza — DISMISSED** [review]
Thank you so much @sanders41!

I let @bidoubiwa review before any merge 🙂

**Comment by bidoubiwa** [comment]
Hey @sanders41 thanks a lot! This is really great information. Unfortunately I could not make it work. Do you have an example config file with localhost that works?

**Review by bidoubiwa — APPROVED** [review]
🔥 Thanks so much for the investigation. This is really great and needed information for our different CI's. 

🔥🔥🔥🔥

File: README.md [hunk 1]
```diff
@@ -459,6 +462,41 @@ If used, `min_indexed_level` is ignored.
 }
 ```
 
+#### `js_render` (optional)
+
+When `js_render` is set to `true`, The scraper will use ChromeDriver. This is needed for pages that are rendered with JavaScript, for example pages generated with React, Vue, or applications that are running in development mode: `autoreload` `watch`.
```
Comment (curquiza) [inline README.md:467]:
```suggestion
When `js_render` is set to `true`, the scraper will use ChromeDriver. This is needed for pages that are rendered with JavaScript, for example, pages generated with React, Vue, or applications that are running in development mode: `autoreload` `watch`. 
```

File: README.md [hunk 2]
```diff
@@ -459,6 +462,41 @@ If used, `min_indexed_level` is ignored.
 }
 ```
 
+#### `js_render` (optional)
+
+When `js_render` is set to `true`, The scraper will use ChromeDriver. This is needed for pages that are rendered with JavaScript, for example pages generated with React, Vue, or applications that are running in development mode: `autoreload` `watch`. 
+
+After installing ChromeDriver, provide the path to the bin using the following environment variable `CHROMEDRIVER_PATH` (default value is `/usr/bin/chromedriver`.
```
Comment (curquiza) [inline README.md:469]:
```suggestion
After installing ChromeDriver, provide the path to the bin using the following environment variable `CHROMEDRIVER_PATH` (default value is `/usr/bin/chromedriver`).
```

**Review by curquiza — CHANGES_REQUESTED** [review]

**Review by bidoubiwa — COMMENTED** [review]
Thanks a lot 🔥 LGTM

**Comment by curquiza** [comment]
Thanks again for your PR @sanders41! 🚀 

bors merge

**Pull request merged**